app/controllers/orchestrations_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# OrchestrationsController allows to handle orchestrations
+class OrchestrationsController < ApplicationController
+  before_action :can_retry?, only: :retry_bootstrap
+
+  def retry_bootstrap
```

Comment:
Would it make sense, to keep a REST style, to change this to the `create` method ?

Reply:
It's not exactly a `create`... as we are requesting a retry. Calling to this endpoint when no previous orchestration exists will fail, also it doesn't make sense to POST on `/orchestrations/:id/retry` or something similar, as we aren't really retrying a specific run, just a specific orchestration type.

Honestly, I'd leave it as it is for now... I know it's not very RESTy...